src/providers/ruby.rs

```diff
@@ -111,7 +111,7 @@ impl RubyProvider {
 
         setup.add_cmd(format!(
             "curl -fsSL https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-installer | bash -s stable \
-            && printf '\\neval \"$(rbenv init -)\"' >> /root/.profile \
+            && printf '\\neval \"$(~/.rbenv/bin/rbenv init -)\"' >> /root/.profile \
```

Comment:
Why did this have to change as well? Looks fine, just curious

Reply:
Apparently this line is launching an interactive shell, so the the updated PATH is not available there yet